paywall/src/data-iframe/Mailbox.ts

```diff
@@ -101,6 +101,8 @@ export default class Mailbox {
    * We can use this to determine whether the user has purchased a key in another tab
    */
   setupStorageListener() {
+    if (!this.useLocalStorageCache) return
+    // TODO: check to see if there are any changes to the keys and don't retrieve if not
```

Comment:
Do you mind linking to the issue/PR here in the comment so we avoid reproducing this?